test/test.js

```diff
@@ -0,0 +1,109 @@
+var nockBack = require('nock').back;
+var should = require('chai').should();
+var tinder = require('./../tinder');
+var client = new tinder.TinderClient();
+var fs = require('fs');
+
+var config = require('./config');
+var TINDER_HOST = 'https://api.gotinder.com';
+nockBack.fixtures = __dirname + '/nockFixtures'; 
+
+if(config.online){
+  nockBack.setMode('wild');
+}else{
+  nockBack.setMode('record');
```

Comment:
Maybe it's better to use `lockdown: use recorded nocks, disables all http calls even when not nocked, doesn't record`. Otherwise, it can still make http calls